src/core/i18n/translationprocessor.js

```diff
@@ -2,46 +2,48 @@ import { getNPlurals, getPluralFunc, hasLang } from 'plural-forms/dist/minimal';
 
 export default class TranslationProcessor {
   /**
-   * Processes a translation which includes performing interpolation, pluralization, or both
-   * @param {string} translations The translations, or a stringified JSON of possible translations
+   * Processes a translation which includes performing interpolation, pluralization, or
+   * both
+   * @param {string | Object} translations The translation, or an object containing
+   * translated plural forms
    * @param {Object} interpolationParams Params to use during interpolation
    * @param {number} count The count associated with the pluralization
+   * @returns {string} The translation with any interpolation or pluralization applied
    */
   static process (translations, interpolationParams, count) {
     const stringToInterpolate = this._selectStringToInterpolate(translations, count);
     return this._interpolate(stringToInterpolate, interpolationParams);
   }
 
   /**
-   * If translations is json, parse it and choose the correct plural form.
-   * Otherwise just return the non-interpolated translation string.
-   * @param {string} translations
+   * If translations is a string, return it
+   * Otherwise select the correct plural form based on count
+   * @param {string | Object } translations
    * @param {number} count
    * @returns {string}
    */
   static _selectStringToInterpolate (translations, count) {
```

Comment:
Small thing, but I'm not sure we need this `_selectStringToInterpolate` method. It's not doing much after removing the `try-catch`, it's almost a pass through to `_selectPluralForm`. What would you think of getting rid of it entirely?  Whenever I see methods that have logic that just returns the initial input, my reaction is that the function should just not be called in that case.

Reply:
Good point! I can take it out